dowhy/causal_identifier/adjustment_set.py

```diff
@@ -0,0 +1,27 @@
+class AdjustmentSet:
```

Comment:
Currently, dowhy stores adjustment sets as a list of dictionaries where the dictionaries will typically look like {"backdoor_set": [...]} (but will occasionally have an additional "num_paths_blocked_by_observed_nodes " field). This means in several places across the repo the raw string "backdoor_set" has to be used to access the actual adjustment set. 

I've added this class so that one can more safely access the adjustment set (eg by calling the getter method) rather than using the raw string, and also so that we can support and delineate new adjustment sets (eg the generalized adjustment set, which generalizes the backdoor criterion)